src/components/FormattedAmount.vue

```diff
@@ -121,8 +131,22 @@ export default class FormattedAmount extends Mixins(NumberFormatterMixin) {
       classes.push(`${baseClass}--shifted`);
     }
 
+    if (this.isValueWider) {
+      classes.push(`${baseClass}--value-wider`);
+    }
+
     return classes.join(' ');
   }
+
+  checkValueVisibility(): void {
+    const { parent, value } = this.$refs;
+    const { offsetWidth: parentWidth } = parent as HTMLElement;
```

Comment:
please be more specific like `HTMLSpanElement`, and add jsdoc here please cuz it's not clear how it works without project run